initializers/logger.js

```diff
@@ -25,16 +26,24 @@ const createGcLogger = logger => ({
 
 		return new Proxy(child, handler);
 	},
-	log: level => function() {
-		if(arguments.length) {
+	// this nested function is left as a un-named function so we have access to the arguments object
+	log: level => function () {
+		if (arguments.length) {
 			const [first, data = {}, meta = {}] = arguments;
-			let message, err;
+			let message;
+			let err;
+			let resData;
 
-			if(typeof first === 'string') {
+			if (typeof first === 'string') {
 				message = first;
 			} else if (first && first.message) {
 				err = first;
-				message = data.supplementalMessage || err.message;
+				if (err.response && err.response.data) {
+					resData = JSON.stringify(err.response.data)
+				} else if (err.response && err.response.body) {
+					resData = `${err.response.statusMessage} ${err.response.body}`
+				}
+				message = `${err.message} ${data.supplementalMessage} ${resData}`;
```

Comment:
this looks like it changes behavior to include both `message` and `supplementalMessage` if both are present. i'm not sure if that's a big deal, but might not be desired behavior

Reply:
in fact, i would say it's not desired behavior, as bunyan will automatically log `err.message`, so this would be repeating that information, could clutter the log line

Reply:
I think JD had told me to make that change, to include both if we have both, @bearfact am I remembering that correctly?

Reply:
yes, supplemental suggest "in addition to", so I would prefer we log both.

Reply:
right, but i'm saying `err.message` already gets logged by bunyan, so we will still be getting both

Reply:
I think I see what you're saying, if we pass an error to the logger, bunyan will log the error.message prop, if it exists, separate from the message var we pass it? I guess previously we would log the message twice then if the supplemental message didn't exist?

Reply:
oh yeah, i think you're right

Reply:
yeah looks like the stdSerializers.err will log the err.message if it exists. I went ahead and removed it from this section.